tests/test_notify_fw_interested_suppliers.py

```diff
@@ -120,18 +94,44 @@ def test_happy_paths(mock_uuid4, run_id, dry_run):
 
     mock_logger = mock.create_autospec(logging.Logger, instance=True)
 
-    assert notify_suppliers_of_new_fw_cq_answers(
-        data_api_client=mock_data_api_client,
-        notify_client=mock_notify_client,
-        notify_template_id="8877eeff",
-        framework_slug="g-cloud-99",
-        dry_run=dry_run,
-        stage="production",
-        logger=mock_logger,
-        run_id=run_id,
-    ) == 0
+    with freeze_time('2000-01-04 01:55:55'):
+        assert notify_fw_interested_suppliers(
+            data_api_client=mock_data_api_client,
+            notify_client=mock_notify_client,
+            notify_template_id="8877eeff",
+            framework_slug="g-cloud-99",
+            dry_run=dry_run,
+            stage="production",
+            logger=mock_logger,
+            run_id=run_id,
+        ) == 0
 
     expected_run_id = str(run_id or "12345678-1234-5678-1234-567812345678")
+    expected_context = {
+        'framework_name': 'G-Cloud 99',
+        'updates_url': 'https://www.digitalmarketplace.service.gov.uk/suppliers/frameworks/g-cloud-99/updates',
+        'framework_dashboard_url': 'https://www.digitalmarketplace.service.gov.uk/suppliers/frameworks/g-cloud-99/',
+        'clarification_questions_closed': 'no',
+        'clarificationsCloseAtUTC_displaytimeformat': '12:00am GMT',
+        'clarificationsCloseAtUTC_dateformat': 'Saturday 1 January 2000',
+        'clarificationsCloseAtUTC_datetimeformat': 'Saturday 1 January 2000 at 12:00am GMT',
+        'clarificationsPublishAtUTC_displaytimeformat': '12:00am GMT',
+        'clarificationsPublishAtUTC_dateformat': 'Sunday 2 January 2000',
+        'clarificationsPublishAtUTC_datetimeformat': 'Sunday 2 January 2000 at 12:00am GMT',
+        'applicationsCloseAtUTC_displaytimeformat': '12:00am GMT',
+        'applicationsCloseAtUTC_dateformat': 'Monday 3 January 2000',
+        'applicationsCloseAtUTC_datetimeformat': 'Monday 3 January 2000 at 12:00am GMT',
+        'intentionToAwardAtUTC_displaytimeformat': '12:00am GMT',
+        'intentionToAwardAtUTC_timetoday': 'Today at 12am GMT',
```

Comment:
Can we test a non-exact time as well? Not sure if that's covered elsewhere

Reply:
Done